docker/release/config/opensearch-dashboards/opensearch-dashboards-docker-entrypoint-1.x.sh

```diff
@@ -15,9 +15,6 @@
 #
 #       --opensearch.startupTimeout=60
 
-# Setup Home Directory
-export OPENSEARCH_DASHBOARDS_HOME=/usr/share/opensearch-dashboards
```

Comment:
I think this depends on `ARG OPENSEARCH_DASHBOARDS_HOME` in Dockerfile. so I added this to ENV at Dockerfile